src/CommunityToolkit.Maui/Views/SemanticOrderView/SemanticOrderView.shared.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+using CommunityToolkit.Maui.Core;
+
+namespace CommunityToolkit.Maui.Views;
+
+/// <inheritdoc cref="ISemanticOrderView"/>
+public class SemanticOrderView : ContentView, ISemanticOrderView
+{
+	/// <summary>
+	/// Backing BindableProperty for the <see cref="ViewOrder"/> property.
+	/// </summary>
+	public static readonly BindableProperty ViewOrderProperty =
+		BindableProperty.Create(nameof(ViewOrder), typeof(IEnumerable), typeof(SemanticOrderView), Enumerable.Empty<View>());
+
+	/// <inheritdoc />
+	public IEnumerable ViewOrder
```

Comment:
How come this `IEnumerable` doesn't have a type?

Should it be `IEnumerable<View>` or `IEnumerable<IView>`?